### PR TITLE
pAI mulebots no longer stuns people

### PR DESCRIPTION
**Pull request by lzimann**
:cl: name: Lzimann
tweak: Mulebots with pAI cards no longer stun people
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
Wasn't killing the problem, not stunning?

**Comment by lzimann** [comment]
the problem is how they can permastun you while dealing a massive damage, and this only affects pAI controlled mulebots, so normal mulebots won't get overnerfed(since they are fine the way they are)

**Comment by coiax** [comment]
So pAIs magically make mulebots non-solid?

**Comment by coiax** [comment]
To be clear, I think pAI+mulebot needs a nerf as well, I've just never heard any solution that I like

**Comment by lzimann** [comment]
we can just remove mulebots pAI, it's an one line change

**Comment by MMMiracles** [comment]
I mean this just makes it so whoever hacks the mulebot has to be actively next to their mobile crusher if they want it to be effective. A hell of a lot better than beforehand.

**Comment by optimumtact** [comment]
@coiax normal non human controlled mulebots are easy to avoid and difficult to control as an antag.

**Comment by Bawhoppen** [comment]
make it stun but do no damage is more fun

**Comment by Core0verload** [comment]
This was already denied 3 or 4 times. @ChangelingRain, close it.

**Comment by MMMiracles** [comment]
Close Core0overload. The pAI nerf must live on

**Comment by kevinz000** [comment]
@optimumtact  AIs have pinpointer control :^)

**Comment by Man-Shroom-** [comment]
could we merge this and also make mulebots like

have a proper control system

**Comment by optimumtact** [comment]
This needs something to explain to the end user why they can't run people over, be it a message at the time they are inserted into the mulebot, or some other method.

**Comment by YakumoChen** [comment]
improve don't remove.

Mulebots are easy as fuck to kill anyway. Just remove the stun.

**Comment by MMMiracles** [comment]
@YakumoChen 
...that is exactly what this PR is doing. Did you completely miss the title, or?

**Comment by YakumoChen** [comment]
I didn't say I disapproved of it

**Comment by Zilenan91** [comment]
So wait they'll still two-shot people it just won't stun them?

Why even make the change? If you get run over once you'll be too slow to run away again.

**Comment by Bawhoppen** [comment]
make it stun them but do no damage

File: code/modules/mob/living/simple_animal/bot/mulebot.dm [hunk 1]
```diff
@@ -737,6 +738,10 @@ var/global/mulebot_count = 0
 		unload(get_dir(loc, A))
 	else
 		..()
+		
+/mob/living/simple_animal/bot/mulebot/insertpai(mob/user, obj/item/device/paicard/card)
+	if(..())
+		src.visible_message("[src] safeties are locked on.")
```
Comment (optimumtact) [inline code/modules/mob/living/simple_animal/bot/mulebot.dm]: src is not necessary

**Pull request merged**